examples/material_gallery/lib/demo/scrolling_techniques_demo.dart

```diff
@@ -0,0 +1,152 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'flexible_space_demo.dart';
+
+class _BarGraphic extends StatelessComponent {
+  _BarGraphic({ this.height, this.color, this.leftText, this.rightText: "" });
```

Comment:
you assume left and right are not null in the build, so you should probably assert that it's not null. Especially left, which has no default.